homeassistant/components/homematicip_cloud/sensor.py

```diff
@@ -123,7 +124,7 @@ async def async_setup_entry(
         async_add_entities(entities)
 
 
-class HomematicipAccesspointDutyCycle(HomematicipGenericEntity):
+class HomematicipAccesspointDutyCycle(SensorEntity, HomematicipGenericEntity):
```

Comment:
It's usually safer to put the platform base class last in the multiple inheritence tree, so that the more integration specific class, eg `HomematicipGenericEntity`, is closer to the child class in the MRO.